### DISCUSSION TITLE
In-app language selector

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16645] [General]
As a user, I want to be able to manually select the language used in the app, so it can be different from the one used by my operating system.

### Why is this important?
It's useful for accessibility to keep language settings in-app independent from the ones in the operating system, especially making the app accessible to people who don't speak english, as not all devices allow the user to change their language from english.

### Requirements:
* Option to select a language
* Option to inherit the language from the OS settings
* Search bar to look for languages

**Comment by ibrkhalil** [comment]
Can we make it default to OS language and indicate in the selector that's the system default and remove the option to inherit OS language ?

**Comment by du82** [comment]
> Can we make it default to OS language and indicate in the selector that's the system default and remove the option to inherit OS language ?

What I meant in my proposal was lets say your OS language is Chinese, and you change the in-app language to English. You should see an option to set it back to your OS default.
I think as you mentioned it should start with the default OS language.
